README.md

```diff
@@ -335,7 +335,7 @@ const PNG_LOADER = combineLoaders([
     {
         loader: 'url',
         query: {
-            limit: 100000
+            limit: 100000,
```

Comment:
@Swelly Did you want to make the changes to remove the other files or should I do this?